<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="updateLog"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="updateAllButton">
      <property name="text">
       <string>Update All</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="connectAllButton">
      <property name="text">
       <string>Connect All</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="disconnectAllButton">
      <property name="text">
       <string>Disconnect All</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="connectLog"/>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="disconnectLog"/>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="configLog"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="configAllButton">
      <property name="text">
       <string>Config All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
